### PR TITLE
Stripe adapter

### PR DESCRIPTION
**Pull request by ilyanaDev**
all the stripe payment handler code (aside from the code in the js) is now pulled behind a set of interfaces

### CODE REVIEW

File: src/DevBetterWeb.Core/Interfaces/IPaymentHandlerSubscriptionDTO.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace DevBetterWeb.Core.Interfaces
+{
+  public interface IPaymentHandlerSubscriptionDTO
```
Comment (ardalis) [inline src/DevBetterWeb.Core/Interfaces/IPaymentHandlerSubscriptionDTO.cs:3]: DTOs don't typically need interfaces. Is there a reason for this one vs just a class?

File: src/DevBetterWeb.Infrastructure/DefaultInfrastructureModule.cs [hunk 2]
```diff
@@ -46,6 +48,16 @@ private void RegisterCommonDependencies(ContainerBuilder builder)
...
 
+    private void RegisterPaymentHandlerDependencies(ContainerBuilder builder)
+    {
+      builder.RegisterType<CustomerService>();
+      builder.RegisterType<PaymentMethodService>();
+      builder.RegisterType<SubscriptionService>();
+      builder.RegisterType<PriceService>();
+      builder.RegisterType<PaymentIntentService>();
+      builder.RegisterType<PaymentMethodService>();
+    }
```
Comment (ardalis) [inline src/DevBetterWeb.Infrastructure/DefaultInfrastructureModule.cs:59]: I like that these are all grouped in one helper method.

**Review by ardalis — APPROVED** [review]
Looks good. My only question is why the interface for the DTO. It's a lot of services and maybe we can reorganize to be not quite so many but otherwise it looks good to me.

**Comment by ilyanaDev** [comment]
fixes #211

**Pull request merged**